bin/run-prod.sh

```diff
@@ -7,13 +7,15 @@
 # look for the required files and fail quickly if it's not there
 STARTUP_FILES=(
     "data/last-run-update_locales"
-    "data/last-run-download_database"
 )
 # If DATABASE_URL is defined, that means we're using Postgres not sqlite.
 # However, if DATABASE_URL is NOT defined, we need to be sure the sqlite DB file
-# is already present at startup
+# is already present at startup and that we've downloaded the latest one
 if [[ -z "$DATABASE_URL" ]]; then
-    STARTUP_FILES+=("data/springfield.db")
+    STARTUP_FILES+=(
+        "data/springfield.db"
```

Comment:
do we still need to check for this?

Reply:
If `DATABASE_URL` is not defined at startup we should probably just not startup and let k8s keep the old pods around.

Reply:
> do we still need to check for [data/springfield.db]?

Good point - as long as we're checking for the db-download sentinel we don't need to check for the DB file

Reply:
I guess I just don't see the point of checking these anymore. I think if the instance needs the DB file and it isn't there it just won't work. We can keep this I guess but it just feels like something that could unexpectedly interfere with startup in some situations.